robottelo/vm_capsule.py

```diff
@@ -238,7 +238,7 @@ def _setup_capsule(self):
                 u'Failed to install satellite-capsule package\n{}'.format(
                     result.stderr)
             )
-        cert_file_path = '/tmp/{0}-certs.tar'.format(self.hostname)
+        cert_file_path = '/root/{0}-certs.tar'.format(self.hostname)
```

Comment:
@jyejare why not /tmp

Reply:
The capsule-certs-generate command advice's to use it from /root/ and we are manipulating or 'sed'ing the capsule-certs-generate commands output, we are using that command to install capsule as it is.

Reply:
Updated my above comment with '**not** manipulating'.

Reply:
+ I dont have hard failing where the certs should be SCPed. So I dont want to raise a bug abt the /tmp or /root.